Automattic/gutenberg
# Fix for #36523: Unable to drag patterns in Safari

## Description

Fixes issue: Unable to insert Patterns using Safari by drag and dropping them.

The problem was caused by the positioning of the iFrame that renders the Pattern preview, which was on top of the draggable div element. Most browsers handled it well, but not Safari. To solve this, a new draggable div element was created and placed on top of the iFrame.

## How has this been tested?
Using Safari:

1. Go to the Site Editor.
2. Open the block inserter from the top bar.
3. Open the Patterns tab bar.
4. Confirm ability to drag a Pattern into the canvas.

Repeated the steps above with Google Chrome, Firefox and Brave browsers.





## Screenshots 

## Types of changes




Bug fix

## Checklist:
